docker/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.8'
+
+# CAUTION: Please ensure you edit necessary values in .env before using this Docker Compose file.
+
+# SYNTAX: docker compose -f docker/docker-compose.yml up --detach
+
+services:
+  app:
+    depends_on:
+      postgres:
+        condition: service_healthy
+    # OPTIONAL: Rebuild cheqd credential-service Docker image, if you want build your own
+    # Default is to pull in the pre-published image on GitHub Container Registry
+    # SYNTAX: docker compose -f docker/docker-compose.yml build
+    # build:
+    #   context: ../
+    #   dockerfile: docker/Dockerfile
+    # CAUTION: Change image section's value if building your own image in section below
+    image: ghcr.io/cheqd/credential-service:staging-latest
+    ports:
+      - 8787:8787
+    env_file:
```

Comment:
this one is not needed cause it by default searches it

Reply:
Seems, it doesn't search it. Instead of that I need to define every env variables in docker-compose.yml: https://docs.docker.com/compose/environment-variables/set-environment-variables/